tethys_gizmos/gizmo_options/cesium_map_view.py

```diff
@@ -330,6 +331,17 @@ class CesiumMapView(TethysGizmoOptions):
                 )
             ]
 
+        **Primitives**
+
+        Support Cesium Primitive such as Cesium Ion
+
+        You can load a Cessium Ion using the following pattern:
+
+        ::
+
+            primitives = [{'Cesium.Cesium3DTileset': {'url': {'Cesium.IonResource.fromAssetId': 512295}}},
+                          {'Cesium.Cesium3DTileset': {'url': {'Cesium.IonResource.fromAssetId': 458785}}}
+                          ]
```

Comment:
Instead of showing with the raw Dictionary, we should show it in terms of the `MVLayer` object support:

```
primitives = [
    MVLayer(
        options={'Cesium.Cesium3DTileset': {'url': {'Cesium.IonResource.fromAssetId': 96188}}},
        data={'layer_name': 'Layer 1', 'layer_variable': 'variable', 'layer_id': 1}
    )
]

Reply:
primitives=[
                {'Cesium_Ion_Data': {'Cesium.Cesium3DTileset': {'url': {'Cesium.IonResource.fromAssetId': 123}}}},
                MVLayer(
                    source='Cesium3D',
                    legend_title='Cesium 3D Buildings',
                    options={'Cesium.Cesium3DTileset': {'url': {'Cesium.IonResource.fromAssetId': 96188}}},
                    data={'layer_name': 'Cesium_Buildings', 'layer_variable': 'variable', 'layer_id': 1}
                )
            ]

Reply:
add support for both cases